Solutions/Microsoft Windows SQL Server Database Audit/Hunting Queries/SQL-UserRoleChanged.yaml

```diff
@@ -1,6 +1,9 @@
 id: 80a420b3-6a97-4b8f-9d86-4b43ee522fb2
 name: User Role altered on SQL Server
 description: |
+    This hunting query identifies user role altered on SQL Server
```

Comment:
"This hunting query identifies user role altered on SQL Server. It relies on the SQLEvent KQL Parser function."